packages/widget-chat/src/actions/user.js

```diff
@@ -37,7 +37,7 @@ export function fetchCurrentUser(spark) {
 export function fetchUser(userEmail, spark) {
   return (dispatch) => {
     dispatch(requestUser(userEmail));
-    spark.user.getByEmail({email: userEmail})
+    spark.user.getUUID(userEmail)
       .then((user) => dispatch(receiveUser(userEmail, user)));
```

Comment:
Without the receiveUser method, this just makes a call and doesn't do anything else. Should we just get rid of this action or do we need this to get the conversation?